CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# [1.5.1](https://github.com/algolia/autocomplete/compare/v1.5.0...v1.5.1) (2021-12-09)
+
+### Bug Fixes
+
+* **concurrency:** ensure panel stays closed after blur ([#829](https://github.com/algolia/autocomplete/issues/829)) ([2dd34e0](https://github.com/algolia/autocomplete/commit/2dd34e0ac1eae19d87105668bd13155b543ca336))
+* **deps:** update dependency preact to v10.6.1 ([#773](https://github.com/algolia/autocomplete/issues/773)) ([12de8bb](https://github.com/algolia/autocomplete/commit/12de8bb21b1847e37e3fd6420c7740e152d2541e))
```

Comment:
Renovate labeled it as a `fix` which is why it's here.

It's a dependency so I guess it makes sense to have it in the changelog, just not sure about it being in "Bug fixes". The only impact could be having two versions of Preact in the codebase.

Reply:
That's in the E2E app so it shouldn't be part of our changelog (https://github.com/algolia/autocomplete/pull/773/files).

Reply:
```suggestion
```

Reply:
True! I didn't notice, thought it was in `autocomplete-js`.